### ISSUE TITLE
Suggestion: Culture origins

### ISSUE THREAD

**Issue by MilesBHuff** [TownyAdvanced/TownyCultures#20]
Configurable origins for cultures, each with a different bonus and malus;  and the bonuses/maluses scale according to the criteria in #18.
There could be one origin per biome, and the bonus/malus a culture receives would match that of the biome in which the culture's first town resides.

For example:  Culture A is created in Town X.  Town X's homeblock is in a desert biome.  So culture A gets the desert bonuses/maluses.

This could be applied retroactively to existing cultures by treating the largest town of that culture as the origin.

**Comment by LlmDl** [comment]
Sounds like an awful lot of work, requiring us to start saving a lot more information about cultures, ie: where they started, what their buffs and debuffs are...

**Comment by MilesBHuff** [comment]
Yeah, it's a huge feature.  The kind of thing I have little hopes of seeing implemented, but which I figured I'd file a request for anyway.
